docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/TextClassification.cs

```diff
@@ -13,6 +13,8 @@ public static class TextClassification
         /// </summary>
         public static void Example()
         {
+            // This will download an unfrozen (SavedModel format) pre-trained
```

Comment:
A note on style: rather than "This will download an unfrozen ...", it is customary to say "Download an unfrozen ...". In general use present tense and make the comments as terse as possible